simtools/applications/validate_schema_files.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python3
+"""
+    Summary
+    -------
+    Validate parameter description files using a json schema file.
+
+    Command line arguments
+    ----------------------
+    file_name (str)
+      input file to be validated
+    schema (str)
+      schema file (jsonschema format) used for validation
+
+    Example
+    -------
+
+    .. code-block:: console
+
+        simtools-validate-schema-file \
+         --file_name tests/resources/MST_mirror_2f_measurements.schema.yml \
+         --schema jsonschema.yml
+
+"""
+
+import logging
+from pathlib import Path
+
+import jsonschema
+import yaml
+
+import simtools.util.general as gen
+from simtools.configuration import configurator
+
+
+def _parse(label, description):
+    """
+    Parse command line configuration
+
+    Parameters
+    ----------
+    label (str)
+        application label
+    description (str)
+        application description
+
+    Returns
+    -------
+    config (Configurator)
+        application configuration
+
+    """
+
+    config = configurator.Configurator(label=label, description=description)
+    config.parser.add_argument("-f", "--file_name", help="file to be validated", required=True)
+    config.parser.add_argument("-s", "--schema", help="json schema file", required=True)
+    return config.initialize()
+
+
+def load_schema(schema_file):
+    """
+    Load parameter schema from file.
+
+    Parameters
+    ----------
+    schema_file (str)
+        schema file
+
+    Returns
+    -------
+    parameter_schema (dict)
+        parameter schema
+
+    Raises
+    ------
+    FileNotFoundError
+        if schema file is not found
+
+    """
+
+    try:
+        with open(schema_file, "r", encoding="utf-8") as file:
+            parameter_schema = yaml.safe_load(file)
+    except FileNotFoundError:
+        logging.error(f"Schema file {schema_file} not found")
+        raise
+
+    return parameter_schema
+
+
+def validate_schema_file(input_file, schema):
+    """
+    Validate parameter file against schema.
+
+    Parameters
+    ----------
+    input_file (str)
+        input file to be validated
+    schema (dict)
+        schema used for validation
+
+    Raises
+    ------
+    FileNotFoundError
+        if input file is not found
+
+    """
+
+    try:
+        with open(input_file, "r", encoding="utf-8") as file:
+            data = yaml.safe_load(file)
+    except FileNotFoundError:
+        logging.error(f"Input file {input_file} not found")
+        raise
+
+    try:
+        jsonschema.validate(data, schema=schema)
+    except jsonschema.exceptions.ValidationError:
+        logging.error(f"Schema validation failed for {input_file} using {schema}")
+        raise
+
+    print(f"Schema validation successful for {input_file}")
```

Comment:
You shouldn't use `print` here but rather the logger. In fact, I just realised that you use the `logging` directly instead of getting a logger. That is not how we've done it elsewhere and I think the format of the logging messages would be different if you do it like that and the level set by the user will not be respected (but I am not sure).

Reply:
Changed this to logger.info(...)